go/oasis-test-runner/scenario/e2e/txsource.go

```diff
@@ -219,6 +237,7 @@ func (sc *txSourceImpl) startWorkload(childEnv *env.Env, errCh chan error, name
 		"--log.format", logFmt.String(),
 		"--log.level", logLevel.String(),
 		"--" + flags.CfgGenesisFile, sc.net.GenesisPath(),
+		"--" + workload.CfgRuntimeID, runtimeID.String(),
```

Comment:
:information_source: `runtimeID` is from common.go, and it's the zero value

Reply:
it's not the zero value: https://github.com/oasislabs/oasis-core/blob/master/go/oasis-test-runner/scenario/e2e/common.go#L115

Reply:
ah thanks for the correction